src/iqstat.cpp

```diff
@@ -208,12 +208,15 @@ Print the main usage/help information.
 void usage() {
     char *msgs[] = {
         "Usage: iqstat [-luvVh] [-u user] [ruleId]",
-        "Show information about your pending iRODS rule executions",
-        "or for the entered user.",
-        " -a        display requests of all users",
-        " -l        for long format",
-        " -u user   for the specified user",
-        " ruleId for the specified rule (implies long format)",
+        "Show information about pending iRODS rule executions (i.e. delayed rules).",
+        "Listings are produced via a general query of the R_RULE_EXEC table.  By default,",
+        "only the invoking user's rules are shown.",
+        " ",
+        "Options are:",
+        " -a        display requests of all users.",
+        " -l        force use of the long format.",
+        " -u user   display requests for the specified user (may not be used together with -a).",
+        " ruleId    limits the query to the specified rule ID (and implies long format).",
```

Comment:
Let's capitalize the first letter of each description.

Reply:
done!